packages/protoplugin-example/README.md

```diff
@@ -25,7 +27,7 @@ npm run build
 
 To run the plugin (i.e. generate files):
 
-`npm run buf:generate`
+`npm run generate`
```

Comment:
For this plugin example, I suggest we use:

```suggestion
`npx buf generate buf.build/bufbuild/eliza`
```

In a project, it makes sense to put the command into an npm script, because you typically want to set it up once and forget. 

But in this example, it would be nice to suggest the user to change the command, and maybe generate other BSR modules, from Git, or from local proto files.

WDYT?

Reply:
That's fine.  I actually had it this way and changed it because I felt putting the URL there would be confusing and that it would have to be explained, which would then inflate the README example.

Reply:
@timostamm Updated.  Lmk if the wording sounds OK.